pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -531,7 +541,7 @@ def _is_bool_const(node: nodes.Return | nodes.Assign) -> bool:
             node.value.value, bool
         )
 
-    def _is_actual_elif(self, node: nodes.If) -> bool:
+    def _is_actual_elif(self, node: nodes.If | nodes.TryExcept) -> bool:
```

Comment:
Is this change necessary?

Reply:
I believe so, because in this line https://github.com/PyCQA/pylint/pull/7888/files/341e7098da296c12fffefbfe43eb4c133426132b#diff-af1d215b95c8b3ac59218ba588186da4aefd936eef4732518196ea309c3cc788R739  a node of either type can land in there

Reply:
👍